frontend/app/components/multi-group.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  classNames: ['tags-group'],
+
+  objects: Ember.computed('scope', function() {
+    return this.store.query(this.get('modelName'), { scope: this.get('scope') });
+  }),
+
+  actions: {
+    clickObj(obj) {
+      this.get('onTagClicked')(obj);
```

Comment:
I would suggest the more generic `onItemClicked` rather than `onTagClicked ` as `Tag` has a specific meaning in Flaredown